include/circt/Dialect/FIRRTL/FIRRTLAnnotations.h

```diff
@@ -478,6 +479,15 @@ struct PortAnnoTarget : public AnnoTarget {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// Utilities for Specific Annotations
+//
+// TODO: Remove these in favor of first-class annotations.
+//===----------------------------------------------------------------------===//
+
+/// Utility for searching for a MarkDUTAnnotation on a module and updating a
```

Comment:
The comment seems to be unfinished sentence ?

Reply:
Thanks! 😅  I'll fix this up in a rebase.